src/content/ar/fundamentals/design-and-ui/responsive/images.md

```diff
@@ -403,18 +513,41 @@ You're a super &#9733;
 
 
 
-أحيانًا تكون أفضل صورة ليست صورة إطلاقًا. ومتى أمكن، استخدم الميزات الأصلية للمتصفح لتوفير الوظائف نفسها أو أخرى شبيهة.  تنشئ المتصفحات مواد مصورة كان يلزمها في السابق توفير صور.   وهذا يعني أن المتصفحات لم تعد بحاجة إلى تنزيل ملفات صور منفصلة وأنها تحجب الصور المعدلة من حيث الحجم تعديلاً غير ملائم.  يمكن عرض الرموز باستخدام ترميز موحد أو خطوط رموز خاصة.
+أحيانًا تكون أفضل صورة ليست صورة إطلاقًا. ومتى أمكن، استخدم الميزات الأصلية للمتصفح لتوفير الوظائف
+نفسها أو أخرى شبيهة. تنشئ المتصفحات مواد مصورة كان يلزمها في السابق توفير صور. وهذا يعني أن
+المتصفحات لم تعد بحاجة إلى تنزيل ملفات صور منفصلة وأنها تحجب الصور المعدلة من حيث الحجم تعديلاً غير ملائم.
+يمكن عرض الرموز باستخدام ترميز موحد أو خطوط رموز خاصة.
 
 
 
 ### وضع النصوص في ترميز، وليست مضمنة في صور
 
-متى أمكن، يجب أن يكون النص نصًا وليس مضمَّنًا داخل صور، كاستخدام الصور في العناوين، أو وضع معلومات الاتصال مثل أرقام الهاتف أو العناوين في الصور مباشرة.  وبذلك لن يتمكن المستخدمون من نسخ المعلومات أو لصقها، وتصبح المعلومات غير متاحة لأجهزة قراءة الشاشة ولن تكون سريعة الاستجابة.  بدلاً من ذلك، ضع النصوص في الترميز واستخدم خطوط الويب عند اللزوم لأرشفة النمط اللازم.
+متى أمكن، يجب أن يكون النص نصًا وليس مضمَّنًا داخل صور، كاستخدام الصور في العناوين، أو وضع معلومات
+الاتصال مثل أرقام الهاتف أو العناوين في الصور مباشرة. وبذلك لن يتمكن المستخدمون من نسخ المعلومات أو
+لصقها، وتصبح المعلومات غير متاحة لأجهزة قراءة الشاشة ولن تكون سريعة الاستجابة. بدلاً من ذلك،
+ضع النصوص في الترميز واستخدم خطوط الويب عند اللزوم لأرشفة النمط اللازم.
 
 ### استخدام  CSS لاستبدال الصور
 
-يمكن للمتصفحات الحديثة استخدام ميزات CSS لإنشاء أنماط كانت من قبل تتطلب صورًا.  على سبيل المثال، يمكن إنشاء التدرجات المعقدة باستخدام العنصر <code>background</code>، كما يمكن إنشاء الظلال باستخدام <code>box-shadow</code>، ويمكن إضافة الجوانب المستديرة باستخدام العنصر<code>border-radius</code>.
-
+يمكن للمتصفحات الحديثة استخدام ميزات CSS لإنشاء أنماط كانت من قبل تتطلب صورًا. على سبيل المثال،
+يمكن إنشاء التدرجات المعقدة باستخدام العنصر `background`، كما يمكن إنشاء الظلال
+باستخدام `box-shadow`، ويمكن إضافة الجوانب المستديرة باستخدام العنصر`border-radius`.
+
+<style>
+  p#noImage {
+    margin-top: 2em;
+    padding: 1em;
+    padding-bottom: 2em;
+    color: white;
+    border-radius: 5px;
+    box-shadow: 5px 5px 4px 0 rgba(9,130,154,0.2);
+    background: linear-gradient(rgba(9, 130, 154, 1), rgba(9, 130, 154, 0.5));
+  }
+
+  p#noImage code {
+    color: rgb(64, 64, 64);
+  }
+</style>
```

Comment:
these styles are copied from the "en" version; seems to be missing: https://github.com/google/WebFundamentals/blame/master/src/content/en/fundamentals/design-and-ui/responsive/images.md#L830-L844